src/polygon/hooks.es

```diff
@@ -0,0 +1,8 @@
+import { useContext, useEffect } from 'react';
+import MapContext from '../map/context';
+
+
+export const useAddBoundsToMap = (area) => {
```

Comment:
`useMapBounds` ? 

`useAddBoundsToMap` sounds a bit like it would return a callback